src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = 40 * days;
+  const withMinDiscount = total - 20;
+  const withMaxDiscount = total - 50;
```

Comment:
[Checklist ](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 1st item, create vairables for main values - 40, 20, 50